src/app/students/stake/create/page.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import React, { useMemo, useRef, useState } from "react";
+import Link from "next/link";
+import InfoIcon from "../../pitch/InfoIcon";
+import JoditEditor from "jodit-react";
+
+export default function Page() {
+  const contentEditor = useRef(null);
+  const stakeEditor = useRef(null);
+  const [content, setContent] = useState("");
+  const [stake, setStake] = useState("");
+  const config = {
```

Comment:
You used useMemo for the config object in src/app/students/mint/create/page.tsx. Should we use it again here or remove it from. src/app/students/mint/create/page.tsx for consistency?